templates/vue-admin-settings.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Adminly Dashboard
```

Comment:
This probably should be changed. Let's decide on the appid and app name later today.

Reply:
hehehe

Reply:
> This probably should be changed. Let's decide on the appid and app name later today.

So as agreed, it can be changed to "files_external_ethswarm"